src/backend/templates/event.hbs

```diff
@@ -173,10 +173,11 @@
         <h1 class="section-header">Organized by {{eventurls.orgname}}</h1><br>
       </div>
       <section class="container">
+        <div class="row">
         <div class="col-sm-12 col-md-12 col-xs-12">
-          <p>
             {{eventurls.orgdescription}}
-          </p>
+            <br><br>
```

Comment:
Instead of adding br tags the design should be adjusted and implemented in the CSS.